### PR TITLE
Add support to extract bins from iOS 18 dyld_shared_cache

### PR DESCRIPTION
**Pull request by donato-fiore**
Note: I didn't test kextex, but I've extracted several bins using dyldex.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Extract Binaries from Apple's Dyld Shared Cache to be useful in a disassembler.
 # Installation
 
 ```
-python3 -m pip install dyldextractor
+pip3 install git+https://github.com/donato-fiore/DyldExtractor.git@iOS-18-Support
```
Comment (jpstotz) [inline README.md:8]: I don't think this change should be included in the PR

Comment (arandomdev) [reply README.md:8]: I agree.

Comment (arandomdev) [reply README.md:8]: Additionally, there are a lot of files that I don't think you intended to include, like `.DS_Store` and the `build` folder.

Comment (donato-fiore) [reply README.md]: I reverted the installation command back to the original and added `.DS_Store` and `build/` to the ignore list

**Pull request merged**